src/coreclr/jit/likelyclass.cpp

```diff
@@ -121,6 +121,11 @@ static unsigned getLikelyClassesOrMethods(LikelyClassMethodRecord*
                                           int32_t                                ilOffset,
                                           bool                                   types)
 {
+    if (maxLikelyClasses == 0)
+    {
+        return 0;
```

Comment:
Could be an assert, but just in case if this API is used with a `JitConfig.X` as a parameter. The logic below relies on `maxLikelyClasses` be > 0.